controllers/api.controllers.js

```diff
@@ -54,3 +54,16 @@ exports.postArticleComment = (request, response, next) => {
 		})
 		.catch(next);
 };
+
+exports.patchArticleById = (request, response, next) => {
+	const { article_id } = request.params;
+	const { inc_votes } = request.body;
+	if (typeof inc_votes !== 'number') {
```

Comment:
Psql will throw an error in this case, no need to check it here. Additionally this type of logic should not be in the controller 🙂